### PR TITLE
Add SslRootCert, Fix SslMode for DB

### PR DESCRIPTION
**Pull request by krcy**
**Description**

Changes proposed in this pull request:

- Add possibility to configure db server's ca cert (sslrootcert)  for  kyma-environment-broker, environments-cleanup-job, subaccount-cleanup-job and trial-cleanup-job
- Fix sslMode configuration

**Related issue(s)**

### CODE REVIEW

**Review by ebensom — APPROVED** [review]

**Review by ebensom — COMMENTED** [review]
Feel free to also bump all these images tags with your PR here, so that the chart will by default use your updated container images:
https://github.com/kyma-project/control-plane/blob/main/resources/kcp/charts/kyma-environment-broker/values.yaml#L3-L14

**Comment by krcy** [comment]
> Feel free to also bump all these images tags with your PR here, so that the chart will by default use your updated container images:

Bumped kyma_environment_broker and job images to PR-2135.

**Review by ebensom — APPROVED** [review]

**Review by ebensom — APPROVED** [review]

**Review by ebensom — DISMISSED** [review]

File: docs/kyma-environment-broker/03-07-environments-cleanup.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ The Environments Cleanup binary allows you to override some configuration parame
 | **APP_DATABASE_HOST** | Specifies the host of the database. | `localhost` |
 | **APP_DATABASE_PORT** | Specifies the port for the database. | `5432` |
 | **APP_DATABASE_NAME** | Specifies the name of the database. | `provisioner` |
-| **APP_DATABASE_SSL_MODE** | Activates the SSL mode for PostgrSQL. See all the possible values [here](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
+| **APP_DATABASE_SSLMODE** | Activates the SSL mode for PostgrSQL. See all the possible values [here](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
```
Comment (NHingerl) [inline docs/kyma-environment-broker/03-07-environments-cleanup.md:27]:
```suggestion
| **APP_DATABASE_SSLMODE** | Activates the SSL mode for PostgrSQL. See [all the possible values](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
```

**Review by NHingerl — APPROVED** [review]

**Review by ebensom — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**